### ISSUE TITLE
Restoring the server from the snapshot

### ISSUE THREAD

**Issue by REDMONDS** [TeaSpeak/TeaSpeak#409]
**Describe the bug**
Restoring the server from the snapshot does not transfer all its settings, and some are incorrect.

**Expected behaviour**
That all server group settings will be correct for the snapshot.

**Screenshots**
Before Create Snapshot: https://prnt.sc/krtvu4 (TeamSpeak 3 Server 3.0.13.8)
After Restore Snapshot: https://prnt.sc/krtskj (TeaSpeak-1.2.15-beta-12)

**Environment information:**
 Operating system: Debian 9.3
 TeaSpeak version: TeaSpeak-1.2.15-beta-12
 I am using `experimental_31`: yes

**Additional context**
After setting these rights manually, they again fly off.

**Comment by REDMONDS** [comment]
@Chazotu In YaTQA, everything is fine, but the client itself is in such trouble))

**Comment by REDMONDS** [comment]
@Chazotu Do you reproduce?

**Comment by REDMONDS** [comment]
@Chazotu The rights are set correctly, looked through YaTQA, but for some reason the client is displayed as a screenshot. In doing so, everything works.

**Comment by WolverinDEV** [comment]
Yeah the clients refuses to display the first n permissions because I added n permissions.
I can't do anything about that, except to build up a own client with a better implementation :)
